site/mkdocs.yml

```diff
@@ -43,7 +43,7 @@ extra:
     Spark: /getting-started/
   versions:
     iceberg: 0.12.1
-    nessie: 0.15.1
+    nessie: 0.17.0
```

Comment:
can we automate bumping this?

Reply:
currently other than this yml file, Nessie version is only present in `version.properties`. 
Based on high level google search I think yml file cannot read version.properties files.
Any suggestions ?

Reply:
probably some handling needs to be added to https://github.com/projectnessie/nessie/blob/3e8f8f6f8ad5ed55972f96335d96c0395c8cda77/.github/workflows/release-create.yml

Reply:
thats in nessie though. Not sure how we can automate this w/o a real stretch.

Reply:
sorry for the confusion on my side. I thought I'm reviewing something in the nessie repo :)